HVZ.Persistence.MongoDB/Repos/GameRepo.cs

```diff
@@ -20,6 +20,10 @@ public class GameRepo : IGameRepo
     public event EventHandler<PlayerRoleChangedEventArgs>? PlayerRoleChanged;
     public event EventHandler<TagEventArgs>? TagLogged;
     public event EventHandler<GameActiveStatusChangedEventArgs>? GameActiveStatusChanged;
+    public event EventHandler<GameUpdatedEventArgs>? GameSettingsChanged;
+    public event EventHandler<OzUpdatedEventArgs>? PlayerJoinedOzPool;
+    public event EventHandler<OzUpdatedEventArgs>? PlayerLeftOzPool;
```

Comment:
How will these be used differently? Would it be simpler to just have a single `OzPoolChanged`?

Reply:
it would simplify front end a bit by removing some conditionals
can be rolled into one if you feel that would work better

Reply:
If there is a real case for having both thats fine then.